python/lsst/source/injection/bin/make_injection_pipeline.py

```diff
@@ -120,6 +120,32 @@ def build_argparser():
         help="Config override for a task, in the format 'label:key=value'.",
         action="append",
     )
+    parser.add_argument(
+        "-ip",
+        "--input-pipelines",
+        type=str,
+        help="""Location(s) of additional input pipeline definition YAML
+        file(s). Tasks from input_pipelines will be added to the injection
+        pipeline.""",
+        required=True,
+        metavar="FILE",
+        nargs="+",
+    )
+    parser.add_argument(
+        "-sn",
+        "--subset_name",
+        type=str,
+        help="""All tasks from input_pipelines will be added to this subset.
+        If the subset does not exist it will be created.""",
+        metavar="FILE",
+    )
+    parser.add_argument(
+        "-d",
+        "--new_subset_description",
```

Comment:
Maybe ``--subset-description`` (also changing `_` to `-`). I see that this argument will only be used if a new subset is created, but I think keeping the name short and making this distinction clear in the description is preferable. E.g.:
```
        help="The description given to a new subset which holds tasks from additional pipelines given above."
        "Note: this argument is ignored if the subset already exists.",
```